src/app/interfaces/available-app.interface.ts

```diff
@@ -10,6 +10,8 @@ export interface AvailableApp {
   description: string;
   app_readme: string;
   app_metadata: AppMetadata;
```

Comment:
Not sure if the `app_metadata` exists on the "available app".
If this type is used in the `app.available` endpoint, it should probably needs to be removed.
I've attached a sample of one app

Reply:
then, let's remove `app_metadata: AppMetadata;` from `AvailableApp`.
It'll not break the app, it's not used anywhere.